feature/foryou/src/main/kotlin/com/google/samples/apps/nowinandroid/feature/foryou/ForYouScreen.kt

```diff
@@ -116,16 +118,22 @@ internal fun ForYouRoute(
     val feedState by viewModel.feedState.collectAsStateWithLifecycle()
     val isSyncing by viewModel.isSyncing.collectAsStateWithLifecycle()
     val deepLinkedUserNewsResource by viewModel.deepLinkedNewsResource.collectAsStateWithLifecycle()
+    val notificationPermissionDialog by viewModel.dialogState.collectAsState()
+    val askNotificationPermission by viewModel.showNotificationPermission.collectAsState()
 
     ForYouScreen(
         isSyncing = isSyncing,
         onboardingUiState = onboardingUiState,
         feedState = feedState,
         deepLinkedUserNewsResource = deepLinkedUserNewsResource,
+        askToShowNotificationPermissionDialog = notificationPermissionDialog,
+        askNotificationPermission = askNotificationPermission,
         onTopicCheckedChanged = viewModel::updateTopicSelection,
         onDeepLinkOpened = viewModel::onDeepLinkOpened,
         onTopicClick = onTopicClick,
         saveFollowedTopics = viewModel::dismissOnboarding,
+        onNotificationPermissionDismiss = viewModel::toggleNotificationPermissionDialog,
```

Comment:
Similar to the comment in the viewmodel, this parameter is named `dismiss`, but it is used as a `toggle` (and called that way later). Could we change this to be two different functions to make the behavior more clear?